### PR TITLE
Section 2 - Kelley McCauley

### PR DESCRIPTION
**Pull request by kelleymccauley**

### CODE REVIEW

File: test/01LessonOneTest.js [hunk 1]
```diff
@@ -5,30 +5,30 @@ var assert = require('assert');
 
 function returnTrue() {
     // should return true
-
+return true;
 }
```
Comment (paulrenenichols) [inline test/01LessonOneTest.js:9]: Good to see you're making progress, @kelleymccauley!

I do tend to indent the code inside functions, though, to make it easier to read.

LIke so:

``` javascript
function returnTrue() {
    // should return true
    return true;
}
```

File: test/01LessonOneTest.js [hunk 2]
```diff
@@ -5,30 +5,30 @@ var assert = require('assert');
...
     // and return false if the first argument is greater than the second argument
-
+return small < big;
 }
 
 function equalTo(a, b) {
     // should return true if the first argument is equal to the second argument
     // and should return true if the first argument is equal to the second argument
-
+r
```
Comment (paulrenenichols) [inline test/01LessonOneTest.js]: It looks like you added a lone `r` here. Need any help to finish these last couple of tests, @kelleymccauley?

Comment (kelleymccauley) [inline test/01LessonOneTest.js]: I don't think so, but thank you! I will submit those shortly and get started on Lesson 01, @paulrenenichols

File: test/01LessonOneTest.js [hunk 3]
```diff
@@ -5,36 +5,37 @@ var assert = require('assert');
...
-
+    return a===b;
 }
 
 function notEqualTo(a, b) {
     // should return true if the first argument is not equal to the second argument
     // and should return false if the first argument is equal to the second argument
 
+    return a!==b;
 }
```
Comment (paulrenenichols) [inline test/01LessonOneTest.js:39]: Awesome! How's Pig Latin going, @kelleymccauley?

Comment (kelleymccauley) [inline test/01LessonOneTest.js:39]: @paulrenenichols, going well, I think. But I cannot figure how out to test my code in node (i.e., how can I act like a user and enter my word into a prompt to be converted to piglatin?). When I put everything into the Chrome console, it looks like it works, but I am not positive until I see it as the user. The only other thing I believe I am missing is the normalization. Just not sure where to put this  - at the beginning or at the end (inside the function, I know).

Comment (kelleymccauley) [inline test/01LessonOneTest.js:39]: @paulrenenichols - just so you don't think I am not reading instructions, I do see the "To run your program, you can run node path/to/PigLatin.js, to run the tests you can run mocha path/to/PigLatin.js." instruction; however, I am not really sure "exactly" how to do this. I do plan on coming tomorrow if I am not able to figure this out tonight though.

Comment (kelleymccauley) [inline test/01LessonOneTest.js:39]: @paulrenenichols - I think I know what to do now - just do the git commit to see if the tests pass. I think :) If I can't figure it out, I will go to the session today.

File: apps/02RockPaperScissors.js [hunk 4]
```diff
@@ -7,10 +8,47 @@ prompt.start();
...
+      if (hand2 === 'paper'){
+        return "Hand one wins!";
+        }
+      }
+
+  if (hand1 ==='scissors'){
+    if (hand2 ==='paper'){
+      return "Hand two wins!";
+      }
+    }
```
Comment (paulrenenichols) [inline apps/02RockPaperScissors.js:40]: @kelleymccauley Your Rock, Paper, Scissors tests pass, but I think this if statement is incorrect here.

If **hand1** is **scissors**, and **hand2** is **paper**, does **hand2** win?

File: apps/03TicTacToe.js [hunk 5]
```diff
@@ -23,24 +23,41 @@ function printBoard() {
...
 function checkForWin() {
-    // Your code here
+    return (horizontalWin() || verticalWin() || diagonalWin())
+  // Your code here
 }
 
 function ticTacToe(row, column) {
-    // Your code here
+  board[row][column] = playerTurn;
+  checkForWin();
```
Comment (paulrenenichols) [inline apps/03TicTacToe.js:54]: Everything looks great! I'm going to give you full credit in the grade book.

One thing, there is a requirement to `console.log()` a message when a player wins.

You could do something like this:

``` javascript
if (checkForWin()) {
  console.log('Player ' + playerTurn + ' Won!')
}
```

File: apps/04TowersOfHanoi.js [hunk 6]
```diff
@@ -16,24 +16,36 @@ function printStacks() {
...
+    else {
+      return false;
+    }
 }
 
 function towersOfHanoi(startStack, endStack) {
-    // Your code here
-
+    movePiece(startStack, endStack);
 }
```
Comment (paulrenenichols) [inline apps/04TowersOfHanoi.js:52]: Also, you need to call **checkForWin()** inside your **towersOfHanoi(startStack, endStack)** to see if the player has won.

**Pull request closed without merging**